src/bundle/Controller/TrashController.php

```diff
@@ -278,13 +283,12 @@ public function restoreAction(Request $request): Response
      *
      * @throws \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
      * @throws \Symfony\Component\Translation\Exception\InvalidArgumentException
-     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
      * @throws \LogicException
      * @throws \InvalidArgumentException
      */
     public function deleteAction(Request $request): Response
     {
-        if (!$this->isGranted(new Attribute('content', 'remove'))) {
+        if (!$this->isGranted(new Attribute('content', 'cleantrash'))) {
```

Comment:
Is this correct?  What if we want user the be able to remove content but not clean the trash?